src/main/java/blusunrize/immersiveengineering/common/util/network/MessageSkyhookSync.java

```diff
@@ -48,7 +49,7 @@ public void fromBytes(ByteBuf buf)
 		entityID = buf.readInt();
 		NBTTagCompound tag = ByteBufUtils.readTag(buf);
 		connection = Connection.readFromNBT(tag);
-		target = new BlockPos(buf.readInt(),buf.readInt(),buf.readInt());
+		target = BlockPos.fromLong(buf.readLong());

```

Comment:
I'd kind of like to keep compatibility with cubic chunks, which this explicitly breaks AFAIK.